Lib/test/support/__init__.py

```diff
@@ -2770,7 +2770,7 @@ def __init__(self):
         import signal
         self.signal = signal
         self.signals = list(range(1, signal.NSIG))
-        # SIGKILL and SIGSTOP signals cannot be ignored nor catched
+        # SIGKILL and SIGSTOP signals cannot be ignored nor caught
```

Comment:
'catched' might be a misspelling of 'cached' rather than 'caught'.  Victor Stinner might be the one who knows.

Reply:
this edit is correct: "caught" was intended.

Reply:
Oops, irregular verb: caught is correct ;-)